cortex/feature_types.py

```diff
@@ -94,9 +98,9 @@ def _wrapper2(*args, **kwargs):
                                    str(kwargs['start']) + '_' +
                                    str(kwargs['end']) + '.cortex')
                     
-                    if os.environ['CORTEX_CACHE_COMPRESSION'] is not None:
-                        assert os.environ['CORTEX_CACHE_COMPRESSION'] is in ['gz', 'bz2', 'lzma', 'zip'], f"Compression method for caching does not exist."
-                        pickle_path += '.' + kwargs.get('compression')
+                    if os.getenv('CORTEX_CACHE_COMPRESSION') is not None:
```

Comment:
Good moves! I've found `os.getenv` to be way easier to use than directly accessing `os.environ`.